### DISCUSSION TITLE
Java stripped spruce

### DISCUSSION THREAD

**Discussion by IdotIcom** [BedrockTweaks/Files#225] [Ideas]
Any plan to add java stripped spruce logs/wood in parity section?
![Screenshot_2024-07-30-11-51-11-21](https://github.com/user-attachments/assets/8c26f7a1-284c-4f9c-b8e8-d5417ee14b2c)

**Comment by SquatchHunter** [comment]
![image](https://github.com/user-attachments/assets/de29349d-9d18-4382-b4e6-aec44a3ffdcc)
I saw in our github, this already existed so I went and had a look
its under retro > Pre 1.18 Stripped Dark

**Reply by IdotIcom** [reply]
?

**Reply by PAPISOP** [reply]
> ![image](https://private-user-images.githubusercontent.com/33571936/355349797-de29349d-9d18-4382-b4e6-aec44a3ffdcc.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjMzOTQ0NzMsIm5iZiI6MTcyMzM5NDE3MywicGF0aCI6Ii8zMzU3MTkzNi8zNTUzNDk3OTctZGUyOTM0OWQtOWQxOC00MzgyLWI0ZTYtYWVjNDRhM2ZmZGNjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MTElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODExVDE2MzYxM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWFhMDk5MTQxNzBhNTBmZDljN2NkNzkxNTVmMzg3YzQzMmI0OTViOGRlZjgyYWU1MmU5ZjQxYjhkYTA1OGUzODQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.gFvaIXi-uOGCnDZi_x-NrA7ELXpi9dHwwC77OxlPyQU) I saw in our github, this already existed so I went and had a look its under retro > Pre 1.18 Stripped Dark

That's dark oak not spruce 💀 @SquatchHunter

**Reply by SquatchHunter** [reply]
my bad, it was late when I posted that lmao

uh, this would be a "if approved by DrAv" situation since its a deviation from VT in this regard.

**Comment by SquatchHunter** [comment]
why'd this close?

**Reply by IdotIcom** [reply]
Mojank already fixed it
![Screenshot_2026-02-01-20-19-34-75](https://github.com/user-attachments/assets/cac3f64c-8d5e-47c5-a571-7aebc7afb8e1)

**Reply by SquatchHunter** [reply]
hm, didnt see that drop lol
guess we could always make a retro pack to bring back the old texture should people want it
